frontend/app/src/locales/es.json

```
@@ -97,41 +97,41 @@
         "error": "Error al cambiar el método de base de costo a {method}: {message}",
         "success": "Método de base de costo cambiado con éxito a {method}"
       },
-      "crypto_to_crypto": "Error setting crypto to crypto: {message}",
+      "crypto_to_crypto": "Error al configurar cripto a cripto: {message}",
       "eth_staking_taxable_after_withdrawal": {
-        "disabled": "Successfully disabled eth staking taxable after withdrawal setting",
-        "enabled": "Successfully enabled eth staking taxable after withdrawal setting",
-        "error": "Failed to change eth staking taxable after withdrawal setting: {message}"
+        "disabled": "Configuración de tributación de staking de ETH deshabilitada correctamente después del retiro",
+        "enabled": "Configuración de tributación de staking de ETH habilitada correctamente después del retiro",
+        "error": "Error al cambiar la configuración de tributación de staking de ETH después del retiro: {message}"
       },
-      "export_csv_formulas": "Error setting the option to include formulas in CSV export: {message}",
-      "gas_costs": "Error setting Ethereum gas costs: {message}",
-      "have_csv_summary": "Error setting the option to include summary in CSV export: {message}",
-      "tax_free": "Tax free period {enabled}",
-      "tax_free_period": "Tax free period set to {period} days"
+      "export_csv_formulas": "Error al configurar la opción para incluir fórmulas en la exportación CSV: {message}",
+      "gas_costs": "Error al configurar los costos de gas de Ethereum: {message}",
+      "have_csv_summary": "Error al configurar la opción para incluir un resumen en la exportación CSV: {message}",
+      "tax_free": "Período livre de tasas {enabled}",
```

```suggestion
      "tax_free": "Período exento de tasas {enabled}",
```

libre es con `b` y no `v`. Se usa `v` en algún pais de america? https://dle.rae.es/libre

---

Yes. Sorry. We use the word 'livre' in Brasil. It's where I come from.